common/djangoapps/entitlements/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+from django.contrib.auth.models import User
+from course_modes.models import CourseMode
+
+
+class CourseEntitlement(models.Model):
+    """
+    Represents a Student's Entitlement to a Course Run for a given Course.
+    """
+
+    user = models.ForeignKey(User)
+
+    # The parent courses UUID
+    course_uuid = models.UUIDField()
+
+    created = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
prefer inheriting from `TimestampModel` rather than implementing this ourselves

Reply:
Just making sure we are I am seeing the correct Model. Did you mean `TimestampedModel`?